internal/core/graphics/image.rs

```diff
@@ -234,6 +234,11 @@ pub enum PixelFormat {
     RgbaPremultiplied,
     /// Alpha map. 8bits. Each pixel is an alpha value. The color is specified separately.
     AlphaMap,
+    /// Distance field. 8bit interpreted as i8.
+    /// The range is such that i8::MIN corresponds to 3 pixels outside of the shape,
+    /// and i8::MAX corresponds to 3 pixels inside the shape.
+    /// The array must be width * height +1 bytes long. (the extra bit is read but never used)
+    SignedDistanceField,
```

Comment:
In light of future multi-channel support, perhaps for color emojis, I wonder if we should call this `SingleChannelSignedDistanceField`? I guess we can rename it later as this is private API.